docs/netlify.toml

```diff
@@ -202,3 +202,103 @@ to = "/docs/rasa/deploy/deploy-rasa/"
 [[redirects]]
 from = "/docs/rasa/action-server/deploy-action-server"
 to = "/docs/rasa/deploy/deploy-action-server/"
+
+[[redirects]]
+from = "/docs/rasa-x/changelog/rasa-x-changelog/"
+to = "/docs/rasa-enterprise/changelog/rasa-enterprise-changelog/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/changelog/rasa-x-changelog/"
+to = "/docs/rasa-enterprise/changelog/rasa-enterprise-changelog/"
+
+[[redirects]]
+from = "/docs/rasa/user-guide/how-to-deploy/"
+to = "/docs/rasa/deploy/deploy-rasa/"
+
+[[redirects]]
+from = "/docs/rasa/how-to-deploy/"
+to = "/docs/rasa/deploy/deploy-rasa/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/api/architecture/"
+to = "/docs/rasa-enterprise/installation-and-setup/architecture/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/api/rasa-x-http-api/"
+to = "/docs/rasa-enterprise/pages/http-api/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/api/rasa-x-http-api/"
+to = "/docs/rasa-enterprise/pages/http-api/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/user-guide/enable-workflows/"
+to = "/docs/rasa-enterprise/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/get-feedback-from-test-users/"
+to = "/docs/rasa-pro/building-assistants/user-satisfaction-survey/"
+
+[[redirects]]
+from = "/docs/studio/user-guide/flow-builder-introduction/"
+to = "/docs/studio/user-guide/flow-builder/introduction/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/installation-and-setup/openshift-kubernetes/"
+to = "/docs/rasa-enterprise/installation-and-setup/installation-guide/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/integrated-version-control/"
+to = "/docs/rasa-enterprise/installation-and-setup/post-deploy-steps/set-up-ivc/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/1.1.x/installation-and-setup/deploy-tools/install/helm-chart-installation/installation.mdx"
+to = "/docs/rasa-enterprise/installation-and-setup/install/helm-chart-installation/installation/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/installation-and-setup/customize/"
+to = "/docs/rasa/installation/installing-rasa-open-source/"
+
+[[redirects]]
+from = "/docs/rasa/training-data-format.mdx"
+to = "/docs/rasa/training-data-format/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/annotate-conversations/"
+to = "/docs/studio/user-guide/annotation/"
+
+[[redirects]]
+from = "/docs/rasa-enterprise/0.24.1/installation-and-setup/integrated-version-control/"
+to = "/docs/rasa-enterprise/installation-and-setup/post-deploy-steps/set-up-ivc/"
+
+[[redirects]]
+from = "/docs/getting-started/demobot/"
+to = "/docs/"
```

Comment:
I think this rule needs to be added to [rasa-website](https://github.com/RasaHQ/rasa-website/blob/main/netlify.toml) for it to work